pkg/controller/servicebinding/naming/naming_test.go

```diff
@@ -0,0 +1,28 @@
+package naming
+
+import "testing"
+
+func TestName(t *testing.T) {
+	data := map[string]interface{}{
+		"service": map[string]interface{}{
+			"name": "database",
+			"kind": "Service",
+		},
+		"name": "db",
+	}
+	envVars := map[string]string{
+		"apikey": "abcdef",
+		"port":   "8888",
+	}
+
+	//strategy := "{{.service.name | upper }}_{{.name | title}}"
+
+	vars, err := Build(data, envVars, "none")
```

Comment:
We need to have tests for all supported naming strategies.

Reply:
added